.github/workflows/benchmark-flow.yml

```diff
@@ -0,0 +1,117 @@
+name: Run a Benchmark Flow
+
+on:
+  workflow_call:
+    inputs:
+      container:
+        type: string
+        default: redisfab/rmbuilder:6.2.7-x64-bullseye
```

Comment:
Why using the 6.2.7 container? WE should use an updated one no?

Reply:
This is the image we currently using. I initially tried to replace it but got into build issues that we will face in a later PR when we remove readies and setups.
As you mentioned and since redisearch doesn't accept versions before 7.2, the image name is misleading but we get and install the right redis version regardless.

Reply:
@alonre24 / @GuyAv46 WRT builder image the important part is to keep Debian “bullseye” image. The redis image used for build/connect is not really the one used in the benchmarks. The benchmarks redis version is controlled by the AWS golden image we've built for the benchmarks and it uses redis 7.2.

Reply:
Why do we need bullseye specifically?